src/coinjoin/client.cpp

```diff
@@ -134,7 +134,7 @@ void CCoinJoinClientQueueManager::ProcessDSQueue(const CNode& peer, CDataStream&
     }
 }
 
-void CCoinJoinClientManager::ProcessMessage(CNode& peer, std::string_view msg_type, CDataStream& vRecv, CConnman& connman)
+void CCoinJoinClientManager::ProcessMessage(CNode& peer, CConnman& connman, std::string_view msg_type, CDataStream& vRecv)
```

Comment:
also don't really see a point here but this one is fine

Reply:
Minor cleanup for subsequent backports involving `net_processing`.